components/raftstore-v2/src/router/response_channel.rs

```diff
@@ -336,9 +339,23 @@ impl WriteCallback for CmdResChannel {
 /// need to be a field of the struct.
 #[derive(Clone, PartialEq, Debug)]
 pub struct ReadResponse {
+    pub response: RaftCmdResponse,
```

Comment:
u64 should be enough for returning a read index.

Reply:
makes sense.